### PR TITLE
Added Jpg to Png Converter Python script

### PR DESCRIPTION
**Pull request by AnuragGupta806**
This Python Script Is a simple GUI that takes JPG image and creates PNG of that file

### CODE REVIEW

File: Python/JPG_to_PNG_Converter.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+def getJPG():
+    '''Function to get image location and open it with pillow'''
+    global im1
+    import_file_path = filedialog.askopenfilename()
+    im1 = Image.open(import_file_path)
+
+
+font = ('helvetica', 12, 'bold')
+bg = 'royalblue'
```
Comment (kaustubhgupta) [inline Python/JPG_to_PNG_Converter.py]: change these variable names

File: Python/JPG_to_PNG_Converter.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if im1 is None:
+        tk.messagebox.showerror("Error", "No File selected")
+    else:
+        export_file_path = filedialog.asksaveasfilename(defaultextension='.png')
+        im1.save(export_file_path)
+
+
+saveAsButton_PNG = tk.Button(text='Convert JPEG to PNG', command=convertToPNG, bg=bg, fg=fg, font=font)      # Convert button
+canvas1.create_window(150, 180, window=saveAsButton_PNG)
+root.mainloop()
```
Comment (kaustubhgupta) [inline Python/JPG_to_PNG_Converter.py]: - The program works perfectly fine but can you add any progress bar or an alert when the file conversion has been completed? 
- Add the screenshot of the program and put both the files (code and image) in a folder under Python Directory

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Python/JPG_to_PNG_Converter.py (hunk 1 above)
Comment (AnuragGupta806) [reply Python/JPG_to_PNG_Converter.py]: i changed these variable names

File: Python/JPG_to_PNG_Converter.py (hunk 2 above)
Comment (AnuragGupta806) [reply Python/JPG_to_PNG_Converter.py]: added an alert after successful image conversion
i didnt get the screenshot part? What exactly?

**Comment by kaustubhgupta** [comment]
Thanks for your contribution! Really liked your script

**Pull request merged**